doc/source/installation/apt_repo.rst

```diff
@@ -9,7 +9,7 @@ of the following distributions:
 
 * Debian 7 ("wheezy")
 * Debian 8 ("jessie")
-* Ubuntu 12.04 LTS (Precise Pangolin)
+* Debian 9 ("stretch")
 * Ubuntu 14.04 LTS (Trusty Tahr)
 * Ubuntu 16.04 LTS (Xenial Xerus)
 * Ubuntu 17.04 (Zesty Zapus)
```

Comment:
we don't support 17.04 anymore,
17.10 (artful) and 18.04 (bionic) are missing from this list